src/router/index.js

```diff
@@ -71,7 +71,46 @@ const routes = [
 
 const router = createRouter({
   history: createWebHashHistory(),
+  base: process.env.FRONTEND,
   routes,
 });
 
+/*
+Router auth logic start
+
+Commented code below that works on `isAuthenticated` state and before every route:
+1. Redirects user to home if user is already logged in and visiting a page that is intended for guest (route.meta.guest)
+2. Redirects user to login if user is not authenticated and visits a page that requires authentication (route.meta.requiresAuth)
+
+Uncomment the lines below when the existing login checks that uses local storage are disabled.
```

Comment:
so, will we do this once the OTP stuff is in place - if needed we can get rid of the local storage one and completely shift to this and then add OTP too!

Reply:
to store the authentication information when the user comes back we will need to use localStorage right?

Reply:
We won't have to use Local storage now. #142 clears some of them and uses vuex state management instead. For remaining ones, let's plan on removing them as a separate PR.

Reply:
<img width="697" alt="Screenshot 2021-04-09 at 1 37 18 PM" src="https://user-images.githubusercontent.com/13185699/114149598-b5aa7e80-9938-11eb-835a-e97fd4fd1590.png">
We don't want to lose the data on refresh right?

Reply:
Yeah, this is fixed in the OTP Pull request. I've added `vue-persistedstate` package. The data is not getting lost upon refresh 👍